ssh/tailssh/tailssh.go

```diff
@@ -411,7 +429,15 @@ func (srv *server) newConn() (*conn, error) {
 		ServerConfigCallback: c.ServerConfig,
 
 		NoClientAuthHandler: c.NoClientAuthCallback,
-		PasswordHandler:     c.fakePasswordHandler,
+
+		// The below two handlers don't actually authenticate, they rely on whatever
+		// authentication was already performed by NoClientAuthCallback. They exist
+		// purely to support buggy clients that aren't satisfied with successful "none"
+		// authentication and insist on supplying a credential.
+		// Note - tailssh_integration_test does not exercise this functionality because
+		// the client used for testing does not exhibit this buggy behavior.
```

Comment:
You can drop "note" ... all comments are notes.

And then add a date, as this is a temporary comment. And a tracking issue.

    // As of 2025-02-10, tailssh_integeration_test doesn't .... See tailscale/tailscale#nnnn.